### PR TITLE
docs(submitting): add more info around the RFC process

### PR DESCRIPTION
**Pull request by plumpy**

### CODE REVIEW

File: community/contributing/submitting.md [hunk 1]
```diff
@@ -17,18 +17,16 @@ issue](https://github.com/spinnaker/spinnaker/issues/new), ping the [#dev
...
-
-Once your design doc is ready, the community will review it and leave feedback.
+It's possible that we'll encourage you to write an RFC if your change is large
+or impactful enough. There is an [RFC
+template](https://github.com/spinnaker/governance/blob/master/rfc/.template.md)
+you should use for this. There are also several [already-submitted
+RFCs](https://github.com/spinnaker/governance/tree/master/rfc) you can use for
+reference.
+
+Once your RFC is ready, add it to [the `spinnaker/governance/rfc`
```
Comment (dorbin) [inline community/contributing/submitting.md]:
```suggestion
When your RFC is ready, add it to [the `spinnaker/governance/rfc`
```

File: community/contributing/submitting.md [hunk 2]
```diff
@@ -17,18 +17,16 @@ issue](https://github.com/spinnaker/spinnaker/issues/new), ping the [#dev
...
+directory](https://github.com/spinnaker/governance/tree/master/rfc) by sending a
+pull request. It will then be reviewed by the community.
```
Comment (dorbin) [inline community/contributing/submitting.md]:
```suggestion
pull request. Members of the community will review it.
```

File: community/contributing/submitting.md [hunk 3]
```diff
@@ -17,18 +17,16 @@ issue](https://github.com/spinnaker/spinnaker/issues/new), ping the [#dev
...
-* How it will integrate with other Spinnaker features.
-* A rough, technical plan of the work required.
-* What integration testing you think is necessary.
-* (Optional) Implementation milestones you plan to hit.
-
-Once your design doc is ready, the community will review it and leave feedback.
+It's possible that we'll encourage you to write an RFC if your change is large
+or impactful enough. There is an [RFC
+template](https://github.com/spinnaker/governance/blob/master/rfc/.template.md)
+you should use for this. There are also several [already-submitted
```
Comment (dorbin) [inline community/contributing/submitting.md]:
```suggestion
to use for this. There are also several [already-submitted
```

**Review by dorbin — APPROVED** [review]
A few small edits, but LGTM otherwise. Thanks @plumpy !

**Review by ezimanyi — APPROVED** [review]

**Review by ajordens — APPROVED** [review]

**Review by robzienert — APPROVED** [review]

**Pull request merged**